source/db/en-projects.json

```diff
@@ -3655,7 +3655,7 @@
     "description": "A safe home for all your data",
     "license_url": "https://github.com/nextcloud/server/blob/master/COPYING-README",
     "logo": "nextcloud.png",
-    "notes": "Nextcloud provides a file sync & share solution that respects your privacy and is fully open source. Sync clients are available for all major platforms. Nextcloud comes with a rich ecosystem of apps including Contacts, Calendar, Video calls, Mail, Newsreader and much more.",
+    "notes": "Nextcloud provides a file sync & share solution that respects your privacy and is fully open source. Sync clients are available for all major platforms. Nextcloud comes with a rich ecosystem of apps including Contacts, Calendar, Video calls, Mail, Newsreader and much more.\n\nNextcloud originated as a fork of ownCloud. It is preferred because ownCloud has an \"open core\" product model and lacks some security features Nextcloud has.",
```

Comment:
I'm not 100% but I'm pretty sure it's markdown. Although inline HTML is allowed in markdown so maybe that could work too. But this is consistent with other project's notes and descriptions.